README.md

```diff
@@ -98,6 +98,39 @@ You can also display a Cupertino style dialog by using the `dialogStyle` paramet
 
 ![image](screenshots/example-cupertino.png)
 
+## Custom Alert Example
+
+You can also display Custom dialog by using the `dialogStyle` parameter.
+
+```dart
+          body: UpgradeAlert(
+          upgrader: Upgrader(
+            debugDisplayAlways: true,
```

Comment:
This example is nice, but it should not include this: `debugDisplayAlways: true,`.

Reply:
Removed ✅